server/middleware/populateCurrentUser.ts

```diff
@@ -45,6 +47,26 @@ export function getUserCaseLoads(userService: UserService): RequestHandler {
   }
 }
 
+async function getActiveCaseload(
+  caseloads: CaseLoad[],
+  userService: UserService,
+  user: Express.User,
+): Promise<CaseLoad | null> {
+  const activeCaseload = caseloads.find(caseload => caseload.currentlyActive)
+  if (activeCaseload) {
+    return activeCaseload
+  }
+  const potentialCaseLoad = caseloads.find(cl => cl.caseLoadId !== '___')
+  if (potentialCaseLoad) {
+    logger.warn(`No active caseload set for user: ${user.username}: setting to ${potentialCaseLoad.caseLoadId}`)
+    await userService.setActiveCaseload(user.token, potentialCaseLoad)
+
+    return potentialCaseLoad
+  }
+
+  return null
```

Comment:
I think I've seen a future job to handle this, not sure it if will ever happen...